frontend/src/components/FileUpload.js

```diff
@@ -1,29 +1,47 @@
 import {fileUpload} from "../services/fileUploadService";
-import {useState} from "react";
 import {useHistory} from "react-router-dom";
+import {IconButton, makeStyles} from "@material-ui/core";
+import AddCircleOutlineOutlinedIcon from "@material-ui/icons/AddCircleOutlineOutlined";
+
+const useStyles = makeStyles((theme) => ({
+    btnChoose: {
+        color: "var(--backgroundColorForms)",
+        bottom: theme.spacing(2),
+        right: theme.spacing(2),
+        position: "fixed"
+    }
+}))
 
 export default function FileUpload({onDraftReceived}) {
 
-    const [file, setFile] = useState(undefined);
-    const history =  useHistory();
+    const history = useHistory();
+    const classes = useStyles();
 
-    const onFormSubmit = (formEvent) => {
-        formEvent.preventDefault();
-        fileUpload(file).then((response) => {
+    const onChange = (changeEvent) => {
+        fileUpload(changeEvent.target.files[0]).then((response) => {
             onDraftReceived(response);
             history.push("/draft");
         })
     }
 
-    const onChange = (changeEvent) => {
-        setFile(changeEvent.target.files[0]);
-    }
-
     return (
-        <form onSubmit={onFormSubmit}>
-            <h3>File Upload</h3>
-            <input type="file" name="file" onChange={onChange} accept="image/*"/>
-            <button type="submit">Upload</button>
-        </form>
+        <div>
+            <label htmlFor="btn-upload">
```

Comment:
should label one element, not three. `htmlFor` is not needed when you wrap an input in a label, but doesn't hurt.

Reply:
@jamarob encompassing all 3 elements is actually needed for the Button to work as intended so they are seen as one element I guess (just tested it without the label and it does not work then). The htmlFor is not needed though, so I removed it.

Reply:
This is interesting 🤔